src/controllers/EventController.ts

```diff
@@ -42,6 +42,7 @@ export class EventController {
 
   @Post('/')
   @ResponseSchema(EventResponse)
+  @OpenAPI({ security: [{ TokenAuth: [] }] })
```

Comment:
What is TokenAuth?

Reply:
it's the security scheme I defined in docs router; time for docs :p